pkg/splunk/enterprise/configuration.go

```diff
@@ -123,10 +123,6 @@ func getSplunkService(cr splcommon.MetaObject, spec *enterpriseApi.CommonSplunkS
 	service.ObjectMeta.Namespace = cr.GetNamespace()
 	instanceIdentifier := cr.GetName()
 	var partOfIdentifier string
-	if instanceType == SplunkMonitoringConsole {
```

Comment:
Could you please clarify why this isn't required anymore?

Reply:
This is for creating the service name for the MC pod. I realized this change should have been done before only. This creates the service name of MC pod as splunk-<namespace>-monitoring-console but what we need is splunk-<crname>-monitoring-console so removed this